manoseimas/settings/base.py

```diff
@@ -154,6 +154,8 @@
     'couchdbkit.ext.django',
     'compressor',
 
+    'manoseimas',                   # depends on: votings
```

Comment:
It still says it depends on votings: https://twitter.com/nzkoz/status/538892801941848064